specification/metadata-3.2.md

```diff
@@ -45,151 +44,161 @@ of `METADATA LIST` MUST be as follows:
-* The 'client' namespace is intended for keys which the user can set and which
-  describe the user's client.
-* The 'ext' namespace is intended for keys which have not been formally
-  registered. Server and client authors are advised that they cannot rely on
-  this namespace to carry any standardized meaning. The main namespaces MAY be
-  replicated under this namespace, except for 'private'.
-* The 'private' namespace is intended for server-internal keys and MUST be seen
-  only with the appropriate server operator permissions. Keys in this
-  namespace are not required to be registered.
+Keys MUST be restricted to the ranges `A-Z`, `a-z`, `0-9`, and `_.:`, and are
+case-insensitive. Values are unrestricted, except that they MUST be UTF-8.
+Server and client authors cannot assume that keys will carry specific
+formatting or meaning.
 
 If a limit is set for keys, it MUST only apply to user-set keys and MUST be
 communicated to the user via `RPL_ISUPPORT` as a numeric value on the
 `METADATA` key.
 
-### Predefined Keys
+## WHOIS
 
-The following keys are predefined for the purposes of the key registry outlined
-above:
+A subset of metadata MAY be sent via the `RPL_WHOISKEYVALUE` event; this
+subset SHALL be set explicitly, rather than informatively or as a side-effect
+of other events. For a complete view of user metadata, see `METADATA LIST`.
 
-| Key            | Meaning                                                      |
-| -------------- | ------------------------------------------------------------ |
-| server.certfp  | SSL certificate fingerprint                                  |
-| user.email     | User's email address                                         |
-| user.phone     | User's phone number                                          |
-| user.website   | User's website                                               |
-| user.im.*      | IM handles; the * is replaced with the relevant service name |
-| user.playing   | Music the user is currently listening to                     |
-| user.status    | The user's current status                                    |
-| client.name    | Client's name                                                |
-| client.version | Client version                                               |
+## IRC Daemons
 
-Except for `server.certfp`, all of these keys SHOULD be considered to be
-free-flowing text with no inherent meaning. `server.certfp` MUST be presented
-as hexadecimal.
+IRC servers may choose to accept or deny any key for any reason, and SHOULD
+implement a blacklist or whitelist functionality for this purpose, configurable
+by the server operators.
 
-## WHOIS
+If `METADATA` is supported, it MUST be specified in `RPL_ISUPPORT` using the
+`METADATA` key.
 
-Metadata MUST appear in `WHOIS` output using `RPL_WHOISKEYVALUE` if any is defined
-for the user.
+## Numerics
 
-## Services
+The numerics 760 through 769 MUST be reserved for metadata, carrying the
```

Comment:
So the idea was to reserve numerics ahead of time to hopefully allow migration to numerics which are grouped by topic.

But, now days it seems we want to just move to informational verbs and away from numerics in general.  So I think just claiming the numerics which are actually defined is good for now, since informational verbs in this way won't be coming for a while.